### PR TITLE
[TACHYON-522] Improve LRUEvictor

### PR DESCRIPTION
**Pull request by uronce-cc**
reopen https://github.com/amplab/tachyon/pull/1022 to master branch.

### CODE REVIEW

**Comment by calvinjia** [comment]
+1 LGTM

File: servers/src/main/java/tachyon/worker/block/meta/StorageDir.java [hunk 1]
```diff
@@ -385,4 +386,8 @@ private void reclaimSpace(long size) {
     }
     return blocksToRemove;
   }
+
+  public BlockStoreLocation toBlockStoreLocation() {
```
Comment (apc999) [inline servers/src/main/java/tachyon/worker/block/meta/StorageDir.java]: Please add unittest to this method in followup PRs

**Comment by apc999** [comment]
(copied from #1022 )
+1 LGTM. A few things that we can address to further clean it up in followup PRs:
(1) May take advantage of StorageDir#getCommittedBytes to simplify your code.
(2) Currently, mMeta means the BlockMetadataManager and meta is used quite often to refer to a BlockMeta instance, a little confusing to me.
(3) of course, unittest. (e.g., unittest of Evictor and added method like toBlockStoreLocation in followup PRs)

**Comment by haoyuan** [comment]
Thanks! Please create JIRAs and fix issues based on the suggestions from @apc999 .

**Comment by uronce-cc** [comment]
sure, work is ongoing :)

**Pull request merged**